src/core/src/helpers.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+if (! function_exists('context')) {
+    /**
+     * Get / set the specified context value in the current coroutine.
+     *
+     * @param  array|string|null  $key
+     * @param  mixed  $default
+     * @param int|null $coroutineId
+     * @return mixed
+     */
+    function context($key = null, $default = null, ?int $coroutineId = null)
+    {
+        return match (true) {
+            is_null($key) => \Hypervel\Context\Context::class,
```

Comment:
The return value here is a little bit weird. Consider returning context instance here, and proxy function calls to static method?